packages/manager/modules/pci/src/projects/project/storages/databases/database/users/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,28 @@
+{
+  "pci_databases_users_title": "Users & Roles",
+  "pci_databases_users_username_label": "Nom d'utilisateur",
+  "pci_databases_users_description_label": "Description",
+  "pci_databases_users_role_label": "Rôles",
+  "pci_databases_users_createdAt_label": "Date de création",
+  "pci_databases_users_status_label": "Statut",
+  "pci_databases_users_status_creating": "Création en cours",
+  "pci_databases_users_status_ok": "Activé",
+  "pci_databases_users_status_deleting": "Suppression en cours",
+  "pci_databases_users_status_deleted": "Supprimé",
+  "pci_databases_users_status_updating": "Mise à jour en cours",
+  "pci_databases_users_status_ready": "Activé",
+  "pci_databases_users_status_error": "En erreur",
+  "pci_databases_users_status_error_inconsistent_spec": "En erreur",
+  "pci_databases_users_status_pending": "En attente",
+
+  "pci_databases_users_modify_password_action_label": "Régénérer le mot de passe",
+  "pci_databases_users_show_key_action_label": "Afficher la clé d'accès",
+  "pci_databases_users_show_cert_action_label": "Afficher le certificat",
+  "pci_databases_users_show_user_informations_action_label": "Afficher les informations",
+
+  "pci_databases_users_password_message_infos": "Le nouveau mot de passe de l'utilisateur {{ user }} est <code>{{ password }}</code>.",
```

Comment:
**question:** Having trouble to find where `pci_databases_users_password_message_*`  translations are used. Are they really needed?

**question:** Would it be possible to remove `<code>` tag from translation by using template concatenation?

Reply:
Those translations are indeed not used anymore, I have removed them, thanks !